code/modules/mob/dead/observer/spells.dm

```diff
@@ -9,22 +9,38 @@ GLOBAL_LIST_INIT(boo_phrases, list(
 	"It feels like someone's standing behind you.",
 ))
 
-/obj/effect/proc_holder/spell/aoe_turf/boo
+/obj/effect/proc_holder/spell/targeted/click/boo
 	name = "Boo!"
 	desc = "Fuck with the living."
+	selection_deactivated_message	= "<span class='shadowling'>Your presence will not be known. For now.</span>"
+	selection_activated_message		= "<span class='shadowling'>You prepare to reach across the veil. <b>Left-click to influence a target!</b></span>"
+	auto_target_single = FALSE
+	allowed_type = /atom // No subtypes are safe from spookage.
 
 	ghost = TRUE
 
 	action_icon_state = "boo"
 	school = "transmutation"
-	charge_max = 600
+	charge_max = 2 MINUTES
 	starts_charged = FALSE
 	clothes_req = 0
 	stat_allowed = 1
 	invocation = ""
 	invocation_type = "none"
-	range = 1 // Or maybe 3?
+	range = 20
+	// no need to spam admins regarding boo casts
+	create_logs = FALSE
 
-/obj/effect/proc_holder/spell/aoe_turf/boo/cast(list/targets, mob/user = usr)
-	for(var/turf/T in targets)
-		T.get_spooked()
+
+/obj/effect/proc_holder/spell/targeted/click/boo/cast(list/targets, mob/user = usr)
+	var/atom/target = targets[1]
+	ASSERT(istype(target))
```

Comment:
I don't *think* you need this because the only clickable things should be atoms, you can't click on datums.

Reply:
Is there a reason not to confirm type safety? Unlikely (Or seemingly impossible), yes, but it's also an inexpensive check.

Reply:
Yeah usually safety checks are good. I just think that its uneeded here because clicking on a non-atom is impossible. But like you said its not expensive, and doesn't matter either way. Up to you (or maints) to decide.